run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+ENV_FILE=".env"
+ENV_EXAMPLE_FILE=".example.env"
+
+if ! [[ -f ${ENV_FILE} ]]
+then
+    echo "Creating ${ENV_FILE} file"
+    cp ${ENV_EXAMPLE_FILE} ${ENV_FILE}
+else
+	required=$(awk -F '=' '{if($2) print $1}' ${ENV_EXAMPLE_FILE} | xargs)
+	current=$(awk -F '=' '{if($2) print $1}' ${ENV_FILE} | xargs)
```

Comment:
Don't know if the pipe to xargs is required, but I'm just going to assume it is.

Reply:
It didn't work out without this pipe, so I will leave it for now, I might remove it later.